test/test_jit.py

```diff
@@ -12000,10 +12010,17 @@ def forward(self):
             pickled_data = archive.read(os.path.join(archive_name, 'attributes.pkl'))
             out = pickle.load(io.BytesIO(pickled_data))
 
-            self.assertEqual(out[0], {"I": "am", "a test": "test"})
-            self.assertEqual(out[1], 2.3)
-            self.assertEqual(out[2], 99)
-            self.assertEqual(out[6], [1, 2, 3, 4])
+            def is_tensor_value(item):
+                if isinstance(item, torch.Tensor):
+                    return True
+                if isinstance(item, list):
+                    return is_tensor_value(item[0])
+                return False
+
+            for loaded_item, item in zip(out, self.get_pickle_values()):
+                if is_tensor_value(item[1]):
```

Comment:
Looks like tensor values are not being tested

Reply:
They're just tensor IDs at this point so they can't be compared

Reply:
is there no way to test this ?

Reply:
It's being tested in the tests that actually load back in a module, whereas this one just loads the attributes pickle file